test/packages/good_v2/kibana/security_ai_prompt/good_v2-security-ai-prompt-1.json

```diff
@@ -0,0 +1,12 @@
+{
+  "attributes": {
```

Comment:
@jsoriano I think Patryk added these v2 and v3 because he was following an example. I don't think we need them since we've only had one version. What do you think, can I delete v2 and v3?

Reply:
Maybe I should be deleting `good` and `good_v2`, according to these tests: https://buildkite.com/elastic/package-spec/builds/1070#0195d331-d5ed-48eb-a080-e698d4ad446c

Could you please clue me into what is wrong here?

Reply:
Correct, this file should be added to the `good_v3` package, but not to packages that don't support this version of the spec.